src/Features/CSharp/Portable/SignatureHelp/ConstructorInitializerSignatureHelpProvider.cs

```diff
@@ -100,9 +100,12 @@ private static bool IsArgumentListToken(ConstructorInitializerSyntax expression,
             var textSpan = SignatureHelpUtilities.GetSignatureHelpSpan(constructorInitializer.ArgumentList);
             var syntaxFacts = document.GetLanguageService<ISyntaxFactsService>();
 
+            var symbolInfo = semanticModel.GetSymbolInfo(constructorInitializer);
```

Comment:
❓ Is no `cancellationToken` available here? Same question applies to several other calls to `GetSymbolInfo` in this PR.